src/components/accordion/accordionMenu.stories.mdx

```diff
@@ -0,0 +1,61 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+
+import AccordionMenuSection from './AccordionMenuSection';
+import AccordionMenu from './AccordionMenu';
+import ListItem from '../list/ListItem';
+import List from '../list';
+
+<Meta
+  title="Patterns/Navigation/Accordion/AccordionMenu"
+  component={AccordionMenu}
+/>
+
+export const Template = (args) => (
+  <AccordionMenu>
+    {args?.items.map((item, i) => (
+      <AccordionMenuSection key={i} title={item.title}>
+        {item.content}
+      </AccordionMenuSection>
+    ))}
+  </AccordionMenu>
+);
+
+# Accordion
+
+<Canvas>
+  <Story
+    name="AccordionMenu (mobile-only)"
+    parameters={{
+      viewport: {
+        defaultViewport: 'sm',
+      },
+    }}
+    args={{
+      allowToggle: true,
+      items: [
+        {
+          title: 'Section 1',
+          content: (
+            <List>
+              <ListItem>Item 1</ListItem>
+              <ListItem>Item 2</ListItem>
+              <ListItem>Item 3</ListItem>
+            </List>
+          ),
+        },
+        {
+          title: 'Section 2',
+          content: (
+            <List>
+              <ListItem>Item 1</ListItem>
+              <ListItem>Item 2</ListItem>
+              <ListItem>Item 3</ListItem>
+            </List>
+          ),
+        },
+      ],
+    }}
```

Comment:
would it be possible to remove the chakra props from the argstable in storybook?